test/bigtest/interactors/loan-history/loan-history-form.js

```diff
@@ -10,20 +12,27 @@ import SelectInteractor from '@folio/stripes-components/lib/Select/tests/interac
 import CalloutInteractor from '@folio/stripes-components/lib/Callout/tests/interactor';
 
   @interactor class LoanHistoryForm {
-    isLoaded = isPresent('#never-radio-button');
+    isLoaded = isPresent('#treatEnabled-checkbox');
 
     whenLoaded() {
       return this.when(() => this.isLoaded);
     }
 
-    clickImmediatelyRadioButton = clickable('#immediately-radio-button');
-    clickIntervalRadioButton = clickable('#interval-radio-button');
-    clickNeverRadioButton = clickable('#never-radio-button');
+    feefineSection = scoped('[data-test-closed-loans-feefine]');
```

Comment:
It looks like there are 2 identical components that contain 3 radio buttons, 1 text field, and 1 combo box. It seems the abstract interactor should be created to prevent code duplication.